.ci.yaml

```diff
@@ -269,7 +269,7 @@ targets:
       cores: "32"
       gcs_goldens_bucket: flutter_logs
       gclient_custom_vars: >-
-        {"download_emsdk": True}
+        {"download_emsdk": true}
```

Comment:
Do you need the double quotes?

Reply:
no, we are passing the string directly to the json parser if we quote it then it will be evaluated as string instead of boolean.